app/views/shared/eligible_region_content_components/_english_language.html.erb

```diff
@@ -1,48 +1,50 @@
+<p class="govuk-body">If you were not born or do not hold citizenship from any of the exempt countries listed, you can provide evidence of your English language ability in one of the following ways:</p>
 
 <ul class="govuk-list govuk-list--number">
   <li>
-    <h3 class="govuk-heading-s">Show you were born or hold citizenship of a country that’s exempt from English language requirements</h3>
-    <p class="govuk-body">You can show this when you upload a copy of your passport or the official identification document for that country.</p>
+    <h3 class="govuk-heading-s">If your teaching qualification or university degree was taught in an exempt country</h3>
+    <p class="govuk-body">Upload the transcript from your teaching qualification or university degree (which you can request from the institution where you studied).</p>
   </li>
 
   <li>
-    <h3 class="govuk-heading-s">Show you were taught in English in a country that’s exempt from English language requirements</h3>
-    <p class="govuk-body">You can show this when you upload the transcript from your degree qualification.</p>
+    <h3 class="govuk-heading-s">If your teaching qualification or university degree was taught completely in English, but in a country that’s NOT exempt</h3>
+    <p class="govuk-body">Upload a document that shows your teaching qualification or university degree was taught exclusively in the English language – this is an official document that you can request from the institution where you studied.</p>
   </li>
 
   <li>
     <h3 class="govuk-heading-s">Show you were taught in English, but in a country that’s NOT exempt from English language requirements</h3>
     <p class="govuk-body">Upload the ‘Medium of instruction’ (MOI) from your degree qualification. You can get this by contacting the institution where you studied.</p>
   </li>
-
-  <li>
-    <h3 class="govuk-heading-s">Show you hold an approved English language certificate</h3>
-    <p class="govuk-body">You’ll need a Secure English Language Test (SELT) at level B2 of the Common European Framework of Reference for Languages (CEFR) scale. This is higher than the B1 proficiency the UK Home Office requires when people apply for UK citizenship, because you’ll be instructing students in English.</p>
-    <p class="govuk-body">The test must be on the list of approved English language tests and awarded within the last 2 years before the date of your application.</p>
+<li>
+  <h3 class="govuk-heading-s">If you have an approved English language certificate</h3>
+  
+  <% unless region.reduced_evidence_accepted%>
+    <p class="govuk-body">You’ll need a Secure English Language Test (SELT) at level B2 from one of these approved providers:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Pearson</li>
+      <li>PSI Services</li>
+      <li>IELTS SELT Consortium</li>
+      <li>LanguageCert</li>
+      <li>Trinity College London</li>    
+    </ul>
```

Comment:
We can get these items from the `EnglishLanguageProvider` models in the database, rather than hard coding the text on this page.